integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -2,6 +2,9 @@
 
 import static com.linkedin.openhouse.javaclient.OpenHouseTableOperations.*;
```

Comment:
for fileio per table, we might also need similar changes in OpenHouseInternalCatalog https://github.com/linkedin/openhouse/blob/main/iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java